paddle/fluid/framework/tensor.h

```diff
@@ -195,8 +217,10 @@ class Tensor {
    *       N,C,H,W for respectively the batch size, the number of
    *       feature maps, the height.
    */
-
-  DataLayout layout_ = DataLayout::kNHWC;
+  // Fix me: here just change the default layout to kNCHW
+  // it doesn't fix the real issue, i.e. feeder should set up tensor layout
+  // according to actual input data
+  DataLayout layout_ = DataLayout::kNCHW;
```

Comment:
Is this acceptable? @luotao1

Reply:
@mozga-intel @tensor-tang 
Discussed with @qingqing01, it's acceptable to modify the default layout.